test/TEST-06-SELINUX/test.sh

```diff
@@ -7,11 +7,16 @@ IMAGE_NAME="selinux"
 TEST_NO_NSPAWN=1
 
 # Requirements:
-# selinux-policy-targeted
+# A selinux policy is installed. Preferably selinux-policy-targeted, but it could work with others
 # selinux-policy-devel
 
-# Check if selinux-policy-devel is installed, and if it isn't bail out early instead of failing
-test -f /usr/share/selinux/devel/include/system/systemd.if || exit 0
+# Check if
+# - selinux-policy-devel is installed and
+# - some selinux policy is installed. To keep this generic just check for the
+#   existence of a directory below /etc/selinux/, indicating a SELinux policy is
+#   installed
+# otherwise t bail out early instead of failing
+test -f /usr/share/selinux/devel/include/system/systemd.if && ls -1p /etc/selinux/ | grep '/' > /dev/null || exit 0
```

Comment:
What about:

```suggestion
test -f /usr/share/selinux/devel/include/system/systemd.if && find /etc/selinux -mindepth 1 -maxdepth 1 -not -empty -type d >/dev/null || exit 0
```

Reply:
still relies on the grep, but I can't find a way to make find exit != 0 otherwise. Since it's a bit more robust I'll commit it